tutorials/XGI in 15 minutes.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>Before wrapping up (or earlier in the tuto), what do you think of just advertising the stats with 2 short examples, and the link to docs/tuto?</p><p> </p><p>Like </p><ul><li>H.nodes.filterby("degree", 2)</li><li>df = H.nodes.multi(["degree", "clustering_coefficient"]).aspandas()</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>